TF_Tooling_Web/src/Connection/MessageReceiver.ts

```diff
@@ -131,7 +131,6 @@ export class MessageReceiver {
         if (this.responseCallbacks !== undefined) {
             for (let key in this.responseCallbacks) {
                 if (key === _response.requestID) {
-                    console.log(_response.message);
```

Comment:
This was added so that the tooling consumer would get a warning if their tooling version did not match, or if their config was in a bad state

Reply:
I see, in that case I need to completely invert this change and *add* it to the MessageReciever in the Settings UI as they were out of sync (which is why I removed this)